clients/src/main/resources/common/message/ListOffsetResponse.json

```diff
@@ -48,7 +48,7 @@
           "about": "The timestamp associated with the returned offset." },
         { "name": "Offset", "type": "int64", "versions": "1+", "default": "-1", "ignorable": false,
           "about": "The returned offset." },
-        { "name": "LeaderEpoch", "type": "int32", "versions": "4+" }
+        { "name": "LeaderEpoch", "type": "int32", "versions": "4+", "default": "-1" }
```

Comment:
Why do we need this default?

Reply:
Until now, this field is an Optional and it's defaulted to `-1` in https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/requests/RequestUtils.java#L30-L32

Otherwise an `int32` default value is `0` with the generator code